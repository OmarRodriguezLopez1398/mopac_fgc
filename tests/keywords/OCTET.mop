UHF MS=3.5
comment1
comment2
Co 0.0 0.0 0.0

UHF OCTET
comment1
comment2
Co 0.0 0.0 0.0

RHF MS=3.5 MECI OPEN(7,7)
comment1
comment2
Co 0.0 0.0 0.0

RHF OCTET MECI OPEN(7,7)
comment1
comment2
Co 0.0 0.0 0.0
