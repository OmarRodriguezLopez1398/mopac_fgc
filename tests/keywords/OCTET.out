 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:36 2021  *
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  3.5
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
UHF MS=3.5
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Co          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    8
                           NO. OF BETA  ELECTRONS =    1

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 UHF MS=3.5
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:36 2021




          FINAL HEAT OF FORMATION =       2654.15927 KCAL/MOL =   11105.00240 KJ/MOL


          COSMO AREA              =         72.38 SQUARE ANGSTROMS
          COSMO VOLUME            =         57.91 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =        -45.222767 EV
          ALPHA SOMO LUMO (EV)    =         45.223 49.375
          BETA  SOMO LUMO (EV)    =        -14.635 -5.167
          NO. OF ALPHA ELECTRONS  =          8
          NO. OF BETA  ELECTRONS  =          1
          MOLECULAR WEIGHT        =         58.9332         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.008 SECONDS
          COMPUTATION TIME        =          0.009 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Co        0.000000000     0.000000000     0.000000000


           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
 -18.86589 -18.69175 -18.69175 -18.25534 -17.65510  -9.78613  45.22277  45.22277
  49.37494


           BETA EIGENVALUES 
 -14.63532  -5.16740  -4.80117  -4.80117  -3.76939   1.89853  52.29768  52.96352
  52.96352


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Co           0.000000        9.0000     1.00000     2.00000     6.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000     0.000     0.000     0.000
 SUM           -0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Co   1.00000   1.00000   1.00000   0.00000   2.00000   1.00000   1.00000   1.00000   1.00000


                    (SZ)    =    3.500000
                    (S**2)  =   15.750000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   x^2-y^2  xz     z^2    yz     xy
         Atomic
    1 Co  7.0000  1.000  1.000  1.000  0.000  0.000  1.000  1.000  1.000  1.000
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:36 2021  *
 *  OCTET      - SPIN STATE DEFINED AS A OCTET
 *  UHF        - UNRESTRICTED HARTREE-FOCK CALCULATION
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *******************************************************************************
UHF OCTET
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Co          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


 OCTET STATE CALCULATION


          UHF CALCULATION, NO. OF ALPHA ELECTRONS =    8
                           NO. OF BETA  ELECTRONS =    1

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 UHF OCTET
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:36 2021




          FINAL HEAT OF FORMATION =       2654.15927 KCAL/MOL =   11105.00240 KJ/MOL


          COSMO AREA              =         72.38 SQUARE ANGSTROMS
          COSMO VOLUME            =         57.91 CUBIC ANGSTROMS

          IONIZATION POTENTIAL    =        -45.222767 EV
          ALPHA SOMO LUMO (EV)    =         45.223 49.375
          BETA  SOMO LUMO (EV)    =        -14.635 -5.167
          NO. OF ALPHA ELECTRONS  =          8
          NO. OF BETA  ELECTRONS  =          1
          MOLECULAR WEIGHT        =         58.9332         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.016 SECONDS
          COMPUTATION TIME        =          0.017 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Co        0.000000000     0.000000000     0.000000000


           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


          ALPHA EIGENVALUES  
 -18.86589 -18.69175 -18.69175 -18.25534 -17.65510  -9.78613  45.22277  45.22277
  49.37494


           BETA EIGENVALUES 
 -14.63532  -5.16740  -4.80117  -4.80117  -3.76939   1.89853  52.29768  52.96352
  52.96352


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Co           0.000000        9.0000     1.00000     2.00000     6.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID        -0.000     0.000     0.000     0.000
 SUM           -0.000     0.000     0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Co   1.00000   1.00000   1.00000   0.00000   2.00000   1.00000   1.00000   1.00000   1.00000


                    (SZ)    =    3.500000
                    (S**2)  =   15.750000


          ATOMIC ORBITAL SPIN POPULATIONS

    Atom   Total    s     px     py     pz   x^2-y^2  xz     z^2    yz     xy
         Atomic
    1 Co  7.0000  1.000  1.000  1.000  0.000  0.000  1.000  1.000  1.000  1.000
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:36 2021  *
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 7 ELECTRONS IN 7 LEVELS
 *  MS=        - IN MECI, MAGNETIC COMPONENT OF SPIN =  3.5
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF MS=3.5 MECI OPEN(7,7)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Co          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  

      THERE ARE    1 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


                       NO. OF SINGLY OCCUPIED LEVELS =    7

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF MS=3.5 MECI OPEN(7,7)
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:36 2021




          FINAL HEAT OF FORMATION =       2654.15927 KCAL/MOL =   11105.00240 KJ/MOL


          COSMO AREA              =         72.38 SQUARE ANGSTROMS
          COSMO VOLUME            =         57.91 CUBIC ANGSTROMS

          NO. OF FILLED LEVELS    =          1
          AND NO. OF OPEN LEVELS  =          7
          MOLECULAR WEIGHT        =         58.9332         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.023 SECONDS
          COMPUTATION TIME        =          0.024 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Co        0.000000000     0.000000000     0.000000000


           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -16.14521 -12.01664 -11.74647 -11.74645 -11.01236  -3.94380  49.09314  49.09315
  50.83631


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Co          -0.000000        9.0000     1.00000     2.00000     6.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000    -0.000     0.000
 SUM            0.000     0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Co   1.00000   1.00000   1.00000   0.00000   2.00000   1.00000   1.00000   1.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  3.5


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         2   3   4   5   6   7   8
          ENERGY

              1        1   1   1   1   1   1   1
        -20.7829       0   0   0   0   0   0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+  -20.782945    0.000000     1+  OCTET       ????

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.
 *******************************************************************************
 ** Cite this program as: MOPAC2016, Version: 21.298M, James J. P. Stewart,   **
 **                           web-site: HTTP://OpenMOPAC.net.                 **
 *******************************************************************************
 **                                                                           **
 **                                MOPAC2016                                  **
 **                                                                           **
 *******************************************************************************

                              PM7 CALCULATION RESULTS

 *******************************************************************************
 *  CALCULATION DONE:                                Fri Oct 29 13:45:36 2021  *
 *  OCTET      - SPIN STATE DEFINED AS A OCTET
 *  RHF        - RESTRICTED HARTREE-FOCK CALCULATION
 *  OPEN(M,N)  - THERE ARE 7 ELECTRONS IN 7 LEVELS
 *  T=         - A TIME OF 172800.0 SECONDS REQUESTED
 *  DUMP=N     - RESTART FILE WRITTEN EVERY 7200.0 SECONDS REQUESTED
 *  MECI       - M.E.C.I. WORKING TO BE PRINTED
 *******************************************************************************
RHF OCTET MECI OPEN(7,7)
 comment1
 comment2
   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *
 


          CARTESIAN COORDINATES 

    NO.       ATOM           X           Y           Z

     1        Co          0.0000      0.0000      0.0000

 General Reference for PM7:
 "Optimization of Parameters for Semiempirical Methods VI: More Modifications to the 
 NDDO Approximations and Re-optimization of Parameters", J. J. P. Stewart, J. Mol. Mod., 1:32, 19 (2013)
 https://link.springer.com/article/10.1007/s00894-012-1667-x

           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


 OCTET STATE CALCULATION

      THERE ARE    1 DOUBLY FILLED LEVELS


      RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS = 1


                       NO. OF SINGLY OCCUPIED LEVELS =    7

          Geometry optimization using EF

     GRADIENT =  0.00000 IS LESS THAN CUTOFF =  1.00000



 -------------------------------------------------------------------------------
 RHF OCTET MECI OPEN(7,7)
 comment1
 comment2


     GRADIENTS WERE INITIALLY ACCEPTABLY SMALL                
     SCF FIELD WAS ACHIEVED                                   


                              PM7 CALCULATION
                                                       MOPAC2016 (Version: 21.298M)
                                                       Fri Oct 29 13:45:36 2021




          FINAL HEAT OF FORMATION =       2654.15927 KCAL/MOL =   11105.00240 KJ/MOL


          COSMO AREA              =         72.38 SQUARE ANGSTROMS
          COSMO VOLUME            =         57.91 CUBIC ANGSTROMS

          NO. OF FILLED LEVELS    =          1
          AND NO. OF OPEN LEVELS  =          7
          MOLECULAR WEIGHT        =         58.9332         POINT GROUP:  R3  


          SCF CALCULATIONS        =          1
 
          WALL-CLOCK TIME         =          0.031 SECONDS
          COMPUTATION TIME        =          0.032 SECONDS




   ATOM   CHEMICAL          X               Y               Z
  NUMBER   SYMBOL      (ANGSTROMS)     (ANGSTROMS)     (ANGSTROMS)
 
     1      Co          0.00000000  *   0.00000000  *   0.00000000  *

                             CARTESIAN COORDINATES

   1   Co        0.000000000     0.000000000     0.000000000


           Empirical Formula: Co  =     1 atoms



      MOLECULAR POINT GROUP   :   R3  


                  EIGENVALUES  
 -16.14521 -12.01664 -11.74647 -11.74645 -11.01236  -3.94380  49.09314  49.09315
  50.83631


              NET ATOMIC CHARGES AND DIPOLE CONTRIBUTIONS

  ATOM NO.   TYPE          CHARGE      No. of ELECS.   s-Pop       p-Pop       d-Pop       
    1         Co          -0.000000        9.0000     1.00000     2.00000     6.00000
 DIPOLE           X         Y         Z       TOTAL
 POINT-CHG.     0.000     0.000     0.000     0.000
 HYBRID         0.000     0.000    -0.000     0.000
 SUM            0.000     0.000    -0.000     0.000


          ATOMIC ORBITAL ELECTRON POPULATIONS

     Atom    s        px        py        pz      x^2-y^2     xz        z^2       yz        xy
    1 Co   1.00000   1.00000   1.00000   0.00000   2.00000   1.00000   1.00000   1.00000   1.00000


          MULTI-ELECTRON CONFIGURATION INTERACTION CALCULATION


          COMPONENT OF SPIN  =  3.5


           NO OF CONFIGURATIONS CONSIDERED = 1

 CONFIGURATIONS CONSIDERED IN C.I.      
 M.O. NUMBER :         2   3   4   5   6   7   8
          ENERGY

              1        1   1   1   1   1   1   1
        -20.7829       0   0   0   0   0   0   0

  STATE       ENERGY (EV)        Q.N.  SPIN   SYMMETRY              POLARIZATION
         ABSOLUTE     RELATIVE                                X           Y           Z

    1+  -20.782945    0.000000     1+  OCTET       ????

           'RELATIVE ENERGY' Is relative to the lowest state calculated.
           This may or may not be the ground state.
           The "+" symbol indicates the root used.

 **********************
 *                    *
 * JOB ENDED NORMALLY *
 *                    *
 **********************



 TOTAL JOB TIME:             0.09 SECONDS

 == MOPAC DONE ==
